### PR TITLE
Replace diagram item properties by attributes

### PR DESCRIPTION
**Pull request by amolenaar**
### PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes

### What is the current behavior?

Some properties of Presentation items are implemented as `reversible_property`, a construct from Gaphas.

Issue Number: #585

### What is the new behavior?

Implement the reversible properties as association and attribute properties, as is used in the Gaphor data model. This will trigger state changes and work nicer with the undo system.

### Does this PR introduce a breaking change?
- [ ] Yes
- [X] No

### CODE REVIEW

File: gaphor/UML/actions/actionspropertypages.py [hunk 1]
```diff
@@ -59,6 +60,7 @@ def _on_ordering_change(self, combo):
     @transactional
     def _on_ordering_show_change(self, button, gparam):
         self.item.show_ordering = button.get_active()
+        print("show_ordering", self.item.show_ordering)
```
Comment (danyeaw) [inline gaphor/UML/actions/actionspropertypages.py:63]: Is this print statement still needed?

File: gaphor/UML/actions/activitynodes.py [hunk 2]
```diff
@@ -179,22 +177,9 @@ def __init__(self, diagram, id=None):
...
-            self._combined = value
-        else:
-            super().load(name, value)
-
-    @observed
-    def _set_combined(self, value):
-        self._combined = value
-
-    combined = reversible_property(lambda s: s._combined, _set_combined)
+    combined: relation_one[UML.ControlNode] = association(
```
Comment (danyeaw) [inline gaphor/UML/actions/activitynodes.py:180]: I like how adding these as associations reduced the number of methods needed :+1:

**Review by danyeaw — APPROVED** [review]
@amolenaar This looks great!

File: gaphor/UML/actions/actionspropertypages.py (hunk 1 above)
Comment (amolenaar) [reply gaphor/UML/actions/actionspropertypages.py:63]: Removed in #588

**Comment by amolenaar** [comment]
Merge #588 instead

**Pull request closed without merging**